steampy/client.py

```diff
@@ -416,3 +416,39 @@ def get_wallet_balance(self, convert_to_decimal: bool = True, on_hold: bool = Fa
             return Decimal(balance_dict[balance_dict_key]) / 100
         else:
             return balance_dict[balance_dict_key]
+
+    @login_required
+    def add_friend(self, steam_id: str) -> bool:
+        response = self._friend_ajax_request(steam_id)
+        return True if response else False
+
+    @login_required
+    def accept_friend(self, steam_id: str) -> bool:
+        return self._friend_ajax_request(steam_id, accept=1)
+
+    _headers = {
+        "Content-Type": "application/x-www-form-urlencoded; charset=UTF-8"
+    }
+
+    def _friend_ajax_request(self, steam_id: str, accept: int = 0) -> dict:
+        url = f'{SteamUrl.COMMUNITY_URL}/actions/AddFriendAjax'
+        data = {
+            "sessionID": self._get_session_id(),
+            "steamid": steam_id,
+            "accept_invite": accept
+        }
+        # returns True or json object if request was succes, otherwise False
+        return self._session.post(url, data=data, headers=self._headers).json()
```

Comment:
Should raise exception if something was wrong, otherwise return datatype should be specified not bool or dict